Prox/Prox/PlaceCarousel/PlaceCarouselViewController.swift

```diff
@@ -101,6 +114,17 @@ class PlaceCarouselViewController: UIViewController {
     // fake the location to Hilton Waikaloa Village, Kona, Hawaii
     fileprivate var fakeLocation: CLLocation = CLLocation(latitude: 19.9263136, longitude: -155.8868328)
 
+    fileprivate var monitoredRegions: [String: GeofenceRegion] = [String: GeofenceRegion]()
+    fileprivate var shouldFetchEvents: Bool {
+        guard var lastLocationFetchTime = timeOfLastLocationUpdate else {
+            return false
+        }
+        let now = Date()
+        lastLocationFetchTime += minimumTimeAtLocationBeforeFetchingEvents
+        return lastLocationFetchTime < now
```

Comment:
nit: it's a little confusing that `lastLocationFetchTime` is actually `timeToFetchEvents` here (since we mutate it). I'd make a new var to store the incremented value with a name like ^.